cmd/minishift/cmd/util/oc.go

```diff
@@ -149,15 +149,15 @@ func getOcPathForProfile() (error, string) {
 
 // PullOpenshiftImageAndCopyOcBinary pull the openshift image if not available and then copy the client binary to Remote/VM machine
 // Any occurring error is also returned.
-func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, requestedOpenShiftVersion string) error {
+func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, requestedOpenShiftVersion string, isGreaterOrEqualToBaseVersion bool) error {
```

Comment:
No need to pass the extra variable, we can decide in the function itself. This way it will more logical IMO.